ubidots/ubidots_in.html

```diff
@@ -94,7 +94,6 @@
             name : {value : ""},
             token : {value : ""},
             device_label : {value : ""},
-            label_device : {value: ""},
```

Comment:
Is it OK to erase this line, I still it being used in `ubidots_in.js`

https://github.com/ubidots/ubidots-nodered/pull/17/files#diff-12fdc83246fbd02b39295019168d2efdR84

Reply:
True, I added it back again.